changelog/1.get_merged_prs.py

```diff
@@ -31,22 +33,50 @@ def parse_isoformat(string):
 else:
     pull_requests = {}
 
+LAST_MODIFIED_DATE = None
+for pr in pull_requests.values():
+    date = parse_isoformat(pr['updated'])
+    if LAST_MODIFIED_DATE is None or date > LAST_MODIFIED_DATE:
+        LAST_MODIFIED_DATE = date
+
+# FIXME: at the moment, because we don't store time zones, to be safe we check
+# for pull requests modified within 24 hours of the cutoff.
+if LAST_MODIFIED_DATE is not None:
+    LAST_MODIFIED_DATE -= timedelta(hours=24)
+
+print("Last modified date: {0}".format(LAST_MODIFIED_DATE))
+
 # We enclose the following code in a try...finally so that if the updating
 # crashes, we still write out what we had.
 
 try:
-    max_issues = repo.get_issues(state='all',sort='created')[0].number
-    i = 0
-    for pr in repo.get_pulls(state='closed', sort='updated', direction='desc', base='master'):
-        i += 1
+
+    # Find the maximum issue number
+    max_issues = repo.get_issues(state='closed', sort='created')[0].number
+
+    # repo.get_pulls doesn't seem to get quite all available pull requests
+    # because state:closed seems to miss some PRs that appear with is:closed
+    # so instead we use get_issues and then get the PR objects manually.
+    # This allows us to also only iterate over issues that have been updated
+    # since a certain time.
+    for i, issue in enumerate(repo.get_issues(since=LAST_MODIFIED_DATE, state='closed')):
```

Comment:
I don't *think* so because it was already doing one request per iteration before to get the labels, so i think the number of requests is the same with this.